sormas-cargoserver/src/main/resources/domain/logback.xml

```diff
@@ -84,7 +84,7 @@
     <logger name="fr.opensagres.xdocreport" level="WARN" />
 
 	<!-- Lower this log level to DEBUG or TRACE to have EJB method invocations logged with duration in ms to find performance leaks -->
-	<logger name="de.symeda.sormas.backend.util.PerformanceLoggingInterceptor" level="INFO" />
+	<logger name="de.symeda.sormas.backend.util.PerformanceLoggingInterceptor" level="TRACE" />
```

Comment:
Should not be checked in on TRACE level, or was this intentional? Letting this run on default on TRACE level is probably too noisy and can cause a performance issue by itself

Reply:
Setting the log level to TRACE was intentional, but still it is debatable whether this should be the default.
The tradeoff is between a slower performance (which might be acceptable since the cargo server is intended for testing and debugging on a local machine, I'm not aware of any usage beyond this) and the necessity to manually adapt this for performance profiling and other occasions where logging on TRACE is helpful.